src/screens/BillChecking/AuthorityMapping/AuthorityMapping.jsx

```diff
@@ -228,6 +227,13 @@ const AuthorityMapping = () => {
       sortable: true,
     },
   ];
+  const [statusData, setstatusData] = useState();
```

Comment:
where useState are suppose to get declared?

Reply:
resolved